packages/@guardian/source-foundations/src/utils/resets.ts

```diff
@@ -16,6 +16,10 @@ const fieldset = `
 	margin: 0;
 `;
 
+/**
+ * @deprecated this uses the SCSS parent selector. In the future we
+ * should look to make this just CSS or move into into our main reset.
+ */
 // remove styling of invalid input elements that gets applied in Firefox
 const input = `
```

Comment:
are we deprecating the name, the language or the functionality here? 

i.e. if this become pure css, what is deprecated about this?

i wonder if we should rename this to rename this to `inputSCSS` and re-export it as a deprecated `input` with the deprecation message?

```suggestion
/**
 * @deprecated use `resets.inputSCSS`
 */
 const input = inputSCSS;
 
// remove styling of invalid input elements that gets applied in Firefox
const inputSCSS = `
```

that effectively adds a new export (`resets.inputSCSS`) while flagging up the old, ambiguous name as something we want to remove.

that would then decouple the misleading name issue from any future work we might want to do on making the reset available as pure css.

Reply:
Agreed, that’s a lot clearer - thanks for the suggestion!

Reply:
As discussed yday - I've changed `input` to just have the type: `SerialisedStyles` which should indicate this properly